iron-fit-behavior.html

```diff
@@ -171,17 +278,14 @@
      * the memoized data.
      */
     resetFit: function() {
-      if (!this._fitInfo || !this._fitInfo.sizedBy.width) {
-        this.sizingTarget.style.maxWidth = '';
+      var info = this._fitInfo || {};
+      for (var property in info.sizerInlineStyle) {
```

Comment:
Uhm, I just didn't want to wrap it all in a `if` statement. 
It doesn't throw an exception if I do `for(var i in {}.some) { console.log(i); }` http://jsbin.com/gapazih/1/edit?html,console